src/libraries/Microsoft.PowerFx.Connectors/ConnectorHelperFunctions.cs

```diff
@@ -147,5 +151,111 @@ internal static string LogException(Exception ex)
         {
             return $"Exception {ex.GetType().FullName}, Message {ex.Message}, Callstack {ex.StackTrace}";
         }
+
+        // DType.ToString() equivalent which is safe for logging (column names are not displayed)
+        internal static string SafeDTypeToString(this DType dType)
```

Comment:
>SafeDTypeToString [](http://example.com/codeflow?start=31&length=17)

This is a nice anonymizer. Should we move it to Core so other processes can call it when needed?

Reply:
I'm good starting in connectors, and moving it ot core when we need it.